ansible/roles/projects/ovn-kubernetes/deploy/tasks/deploy-objects.yaml

```diff
@@ -1,10 +1,9 @@
 ---
 - name: Deploy {{ deployment_object }}
-  k8s:
-    src: "{{ project_dir }}/dist/yaml/{{ deployment_object }}.yaml"
-    state: present
-    kubeconfig: "{{ kubeconfig }}"
-    context: "{{ k8s_context }}"
+  vars:
+    kubernetes_object_file: "{{ project_dir }}/dist/yaml/{{ deployment_object }}.yaml"
+  include_role:
+    name: projects-common/deploy-kubernetes-object-from-file
```

Comment:
This does not seems to be related to the commit it was added on (deploy multus). I believe this should be moved to its relevant commit

Reply:
Since this task will be used commonly, I moved it to a common role then used it to old one (ovn-kubernets) and the new one need it multus deployment.